common/config/rush/pnpm-lock.yaml

```diff
@@ -8682,7 +8655,7 @@ packages:
   /@types/cpx/1.5.0:
     resolution: {integrity: sha512-kuGK3lZqEvHTSDbJcaA6tcPoEXV4/e88YrltZMcQUewZhzYQwNSTMGIiPBqeeFd4LCBo1CX5U6CV6LaHG3wXSg==}
     dependencies:
-      '@types/node': 16.11.7
+      '@types/node': 14.14.31
```

Comment:
?

Reply:
for some reason rush pulled in an older types/node ver, im assuming the types/cpx pkg used * or something like that and pulled in whatever. updated so they all resolve to the same ver in the monorepo